### ISSUE TITLE
Available aggregation types do not take into account the linked tables

### ISSUE THREAD

**Issue by jAlpedrinha** [metabase/metabase#4489]
I am trying to make a `SUM` aggregation on a field that is on a linked table, but the `Sum of` Aggregation options is not shown. I'm assuming this is due to not having a numeric metric on my main table.

As an example, what I have is something like: 

```sql
CREATE TABLE A {
   id UUID,
   created_at datetime,
   details text,
   b_id UUID,
   c_id UUID
}

CREATE TABLE B {
   id UUID
   minutes integer
}

CREATE TABLE C {
   id UUID
   country text
}
```

I want to be able to group by `C`.`country` and aggregate on `SUM(B.minutes)`, which means I have to use `A` as the table I'm questioning, but it doesn't show the `Sum of` on aggregation options. 

Is this somehow a bug, a missing feature, a design option or am I missing something ?

**Comment by salsakran** [comment]
This is a bug, or more accurately an oversight in the logic that determines which aggregation options to display.

**Comment by attekei** [comment]
Fixed by #5590
